auth/src/test/java/org/aerogear/mobile/auth/user/UserPrincipalImplTest.java

```diff
@@ -46,6 +43,14 @@ public void testHasRealmRoleSucceeds(){
         assertEquals(userPrincipalImpl.hasRealmRole("rRole"), true);
     }
 
+    @Test
+    public void testToStringImpl(){
+        assertEquals(
+            "UserPrincipalImpl{username='test-user', email='null', roles=[rRole, cRole]}",
```

Comment:
@rachael-oregan when adding this test to the existing code base... the `roles=...` part was empty